### PR TITLE
[PBW 4988] Updated background style to control bar.

### PR DESCRIPTION
**Pull request by Jgkelle26**
Part one of 2

### CODE REVIEW

File: scss/base/_variables.scss [hunk 1]
```diff
@@ -384,4 +384,7 @@ $slider-volume-thumb-diameter-small: $slider-volume-thumb-diameter * $xsmall-mul
 //bitrate
 $bitrate-horizontal-height: 100px !default;
 
-$font-path: "assets/fonts";
\ No newline at end of file
+$font-path: "assets/fonts";
+
+/gradients
```
Comment (mistermendez) [inline scss/base/_variables.scss]: double slash for comment
```
//gradients
```

Comment (rchalooyala) [inline scss/base/_variables.scss]: this is so cute :)

**Comment by mistermendez** [comment]
also PR w/ 'master' branch

File: scss/base/_variables.scss (hunk 1 above)
Comment (Jgkelle26) [inline scss/base/_variables.scss]: I feel being mildly condescending is not awesome. I could always create a million tickets and have them prioritized so you can do them @rchalooyala  ; )

**Pull request closed without merging**